arangod/RestServer/QueryRegistryFeature.cpp

```diff
@@ -301,6 +302,13 @@ void QueryRegistryFeature::collectOptions(std::shared_ptr<ProgramOptions> option
   options->addOption("--query.optimizer-max-plans",
                      "maximum number of query plans to create for a query",
                      new UInt64Parameter(&_maxQueryPlans));
+                     
+options->addOption("--query.max-nodes-per-callstack",
+                     "maximum number execution nodes on the callstack before "
+                     "splitting the remaining nodes into a separate thread",
+                     new UInt64Parameter(&_maxNodesPerCallstack),
+                     arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden))
+                     .setIntroducedIn(30900);
```

Comment:
Will this be backported to 3.8 or older?

Reply:
No, at least not for now.